tests/std/tests/VSO_0000000_regex_use/test.cpp

```diff
@@ -899,6 +899,36 @@ void test_gh_993() {
     }
 }
 
+void test_gh_997() {
+    // GH-997: <regex>: Grouping within repetition causes regex stack error
+    // GH-1528: <regex>: regex_match gets caught in recursive loop until stack overflow occurs
+
+    try {
+        (void) regex_match(std::string(1025, 'a'), regex("(?:a)+"));
+        assert(false); // adjust test when matching succeeds
+    } catch (const regex_error& ex) {
+        assert(ex.code() == error_stack);
+    }
+
+    try {
+        wregex rgx(L"^http[s]?\\:\\/\\/([^.]+\\.)*github\\.com\\/.*$", icase);

```

Comment:
I recommend using raw string literals whenever possible, to avoid having to mentally reverse C++ string literal escapes.